src/constants.js

```diff
@@ -1,4 +1,5 @@
-import {themeGet, get as getKey} from 'styled-system'
+import {get as getKey} from 'styled-system'
+import themeGet from '@styled-system/theme-get'
 import {theme} from '@primer/components'
 
 export const get = key => themeGet(key, getKey(theme, key))
```

Comment:
Should we export the `get` function from `@primer/components` so we don't have to redefine it?